### ISSUE TITLE
Additional type for Study Docs

### ISSUE THREAD

**Issue by markusniese** [amiv-eth/amivapi#420]
After the decision at the last GV we need an additional type for study docs named "Prüfungsprotokolle" ("exam protocols").
Additionally, the type "PVK-Unterlagen" ("PVK-material") would be helpful.

**Comment by NotSpecial** [comment]
We already have a field for exam protocols etc., named `exams`. A quick check on the website confirms that this is already used for exam protocols. Or do I misunderstand the intentions? How would `exam protocols` differ from the existing `exams`?)

Aside from that, adding a `pvk` key should be no problem. However, I feel that this is a bit too narrow. What about other courses that AMIV offers, e.g. the math intro for new students (if we are still doing that?) -- I feel these documents would fit into the same category as PVK material, but are not PVKs themselves. Maybe we could use a more generic name that could cover all documents from courses hosted by AMIV? Any idea how we could call them?

(I'm a bit hestitant to just put something as specific as `pvk` in because changing the keys later is quite a pain. So we should consider carefully which keys we want to use).

**Comment by markusniese** [comment]
To the first one. I thought a separate category for `exam protocols` would be beneficial as they are very different in nature to written exams for which the current `exams` category is mostly used.
I would see the difference here: `exams` are written where there is just the exam, while `exam protocols` is a recollection with comments from an oral exam. The idea was to split that.

To the second one: A more generic name would be fine, but it should still be unambiguous. The problem currently is that they land all over the place. I've seen them in all four categories as people apparently do not know where to put them and we have to replace them manually.

**Comment by NotSpecial** [comment]
Regarding exams:

**Comment by markusniese** [comment]
### Protocols
So my main reason is maintenance. Having two separate types would make that easier. Mainly because one could cleanly split up the task between the person that deals with protocols and another person that deals with anything else, which is the plan. While that is possible anyway, separate categories would make that more convenient.

Additionally, one could browse through protocols of different subjects which would help us to identify how many protocols are available in which subject, what quality they have and whether they are outdated or not.

### PVK
I agree that the naming of the key is irrelevant, so a more generic version is better. Maybe something like `student documents` or `student prepared documents` (maybe too wordy as well), to make the distinction from `lecture documents`, that already exists.

**Comment by NotSpecial** [comment]
# Exam Protocols

I see the point about easier maintenance. However, as soon as we introduce the new category, we'll have a bit of a mess on our hands as the existing oral exams are still filed under `exams`. Would it be okay for you to take the responsibility to organize that the existing oral exams are correctly reclassified, @markusniese? (As soon as both the API and website support it, of course)

# PVK

How about `study guide`? This could include any kind of document that helps with studying (and goes beyond the existing `cheat sheet`. However, I suppose at the moment, it would mainly be used for PVKs.

**Comment by markusniese** [comment]
### Exam Protocols
Yes, I will take care of reclassifying, as soon as it is active on the website.

### PVK
Sounds good as well.
